Sming/Services/WebHelpers/base64.h

```diff
@@ -1,31 +1,29 @@
-/* base64.h : base-64 / MIME encode/decode */
-/* PUBLIC DOMAIN - Jon Mayo - November 13, 2003 */
-/* $Id: base64.h 128 2007-04-20 08:20:40Z orange $ */
+/*
```

Comment:
Please replace the beginning comment with the standard comment that we have. Something like this will be enough:

```c
/****
 * Sming Framework Project - Open Source framework for high efficiency native ESP8266 development.
 * Created 2015 by Skurydin Alexey
 * http://github.com/SmingHub/Sming
 * All files of the Sming Core are provided under the LGPL v3 license.
 *
 *
 * @author: 2018 - Mikee47 <your@email>
 *
 ****/
```